docker/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Docker
+
+## To compile source using a docker container:
+```
+docker/build-artifacts.sh <platform> <version_string> [source-tarball] <output-directory>
+# e.g.  docker/build-artifacts.sh ubuntu14.04 testbuild ~/heron-release
+```
+
+## To build docker containers for running heron daemons:
+```
+docker/build-docker.sh <platform> <version_string> <output-directory>
+# e.g. docker/build-docker.sh ubuntu14.04 testbuild ~/heron-release
+```
+
+### To run docker containers for local dev work:
+```
+cd docker
+./start-docker.sh heron:<version_string>-<platform>
+# e.g. docker/start-docker.sh heron:testbuild-ubuntu14.04
```

Comment:
If you cd into the docker directory then this command will not work should be `./start-docker` in your example